app/scripts/modules/core/utils/renderIfFeature.directive.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+import {DirectiveFactory} from './tsDecorators/directiveFactoryDecorator';
+
+class RenderIfFeatureDirectiveController {
+  public $attrs: ng.IAttributes;
+  public $element: JQuery;
+
+  constructor(private settings: any) {}
+
+  public initialize(): void {
+    let feature: string = this.$attrs['renderIfFeature'];
+    if (feature && !this.settings.feature[feature]) {
+      this.$element.addClass('ng-hide');
```

Comment:
just a thought, this strikes me as a strange way to hide the element.  can we not just use the `hidden` class, or one of the others, from bootstrap?